File: LuaLibs/src/MyLua/Libs/FrameHandle/ITickedObject.lua
require "MyLua.Libs.Core.GlobalNS"
require "MyLua.Libs.Core.Class"
require "MyLua.Libs.Core.GObject"

local M = GlobalNS.Class(GlobalNS.GObject:new())
M.clsName = "ITickedObject"
GlobalNS[M.clsName] = M

function M:onTick(delta)

end

return M

File: PyLibs/src/Libs/FrameHandle/ITickedObject.py
# -*- coding: utf-8 -*-

from Libs.Core.GObject import GObject;

class ITickedObject(GObject):
    def __init__(self):
        super(ITickedObject, self).__init__();
        
        self.mTypeId = "ITickedObject";

        
    def onTick(self, delta):
        pass;


